src/events_block_comment_resize.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Blockly from "blockly/core";
+import { BlockCommentBase } from "./events_block_comment_base.js";
+
+class BlockCommentResize extends BlockCommentBase {
+  constructor(opt_blockComment, oldSize, newSize) {
+    super(opt_blockComment);
+    this.type = "block_comment_resize";
+    this.oldSize = oldSize;
+    this.newSize = newSize;
+  }
+
+  toJson() {
+    return {
+      ...super.toJson(),
+      newSize: {
+        width: this.newSize.width,
+        height: this.newSize.height,
+      },
```

Comment:
Similarly here, `newSize: this.newSize`?

Reply:
Done